setup.py

```diff
@@ -242,12 +242,29 @@ def build_extensions(self):
             self.copy_file(src, dst)
 
 
+class mypy_type_check(setuptools.Command):
+    description = 'Run MyPy type checker'
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        """Run command."""
+        returncode = subprocess.call(["python", "./tools/mypy-onnx.py"])
```

Comment:
1. Use `sys.executable` instead of `"python"` (this is not nit, because e.g. virtualenv can be used without activation, and so the correct python is actually not in `$PATH`, and also as we recently discovered, many users can have `python3`, `python3.6` as their main python installation).
2. Use absolute path for the `mypy-onnx.py`, otherwise it requires user to invoke from the onnx top directory.